app/services/open_library_service.rb

```diff
@@ -0,0 +1,34 @@
+class OpenLibraryService
+  def initialize; end
+
+  def book_info(param)
+    response = HTTParty.get("https://openlibrary.org/api/books?bibkeys=ISBN:#{param}&format=json&jscmd=data")
+    ParsedResponse.new(param, response.parsed_response)
+  end
+end
+
+class ParsedResponse
```

Comment:
Maybe we could extract this to another file and change the name to something more specific related to OpenLibrary

Reply:
What about add an `as_json` method to return the structure we want?